andela/ah-frontend-valkyrie
# Revert-  Articles CRUD

This reverts commit 62b89b9cf7e6b91503313ab7d84767d663bd8f4b.

**What does this PR do?**

Setup Redux in the application

**Description of Task to be completed?**

- After cloning the repo, `cd` into it and run `npm install` to install dependencies.
- Install the redux dev tools extension in your browser.

**How should this be manually tested?**

- Run `npm start` to run the application.
- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**Any background context you want to provide?**

- For redux to work with react, remember to install react-redux by running `npm install react-redux`.

**What are the relevant pivotal tracker stories?**

[#162633452](https://www.pivotaltracker.com/story/show/162633452)
